test/built-ins/Symbol/asyncDispose/cross-realm.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
```

Comment:
We should not add new tests with `esid: pending`. I guess the relevant sections haven't been added to the spec yet (https://github.com/tc39/proposal-explicit-resource-management/issues/185), but let's just assume they'll be called `sec-symbol.asyncdispose` and `sec-symbol.dispose`.

Reply:
Understood i guess the sections are there now, lemme check!